master/networking/migrate-pools.md

```diff
@@ -0,0 +1,198 @@
+
+  <pre>
+     "ipam": {
+           "type": "calico-ipam"
+      }, 
+  </pre>
+
+   If the type is “calico-ipam,” you are good to go. If the IPAM is set to something else, or the 10-calico.conflist file does not exist, you cannot use this feature in your cluster. 
+
+- Your orchestrator must support changing the pod network CIDR.   
+  Although Kubernetes supports changing the pod network CIDR, not all orchestrators do. For example, OpenShift does not support this feature as described in [`osm_cluster_network_cidr configuration`](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables). Check your orchestrator documentation to verify. 
+
+### How to
+
+#### Migrate from one IP pool to another
+
+Follow these steps to migrate pods from one IP pool to another pool. 
+
+> **Important!** If you follow these steps, existing pod connectivity will not be affected. (If you delete the old IP pool before you create and verify the new pool, existing pods will be affected.) When pods are deleted, applications may be temporarily unavailable (depending on the type of application); plan accordingly. 
+{: .alert .alert-danger }
+
+1. Add a new IP pool.  
+   **Note**: The new IP pool must be within the same cluster CIDR.
+1. Disable the old IP pool.  
+   **Note**: Disabling an IP pool only prevents new IP address allocations; it does not affect the networking of existing pods.
+1. Delete pods from the old IP pool.  
+   This includes any new pods that may have been created with the old IP pool prior to disabling the pool.
+1. Verify that new pods get an address from the new IP pool.
+1. Delete the old IP pool.
+
+### Tutorial
+
+In the following example, we created a Kubernetes cluster using **kubeadm**. When we installed calico nodes in this cluster, a default IP pool was assigned: **192.168.0.0/16**. Now, we want pods in the cluster to use IPs in the CIDR: **10.0.0.0/16** (within the cluster CIDR). 
+
+Let’s run `calicoctl get ippool -o wide` to see the default IP pool, **default-ipv4-ippool**.
+
+<pre>
+NAME                  CIDR             NAT    IPIPMODE   VXLANMODE   DISABLED
+default-ipv4-ippool   192.168.0.0/16   true   Always     Never       false
+</pre>
+
+When we run `calicoctl get wep --all-namespaces`, we see that a pod is created using the default range (192.168.52.130/32).
+
+<pre>
+NAMESPACE     WORKLOAD                   NODE      NETWORKS            INTERFACE
+kube-system   coredns-6f4fd4bdf-8q7zp   vagrant   192.168.52.130/32   cali800a63073ed
+</pre>
+
+Let’s get started changing this pod to the new IP pool (10.0.0.0/16).
+
+#### Step 1: Add a new IP pool
+
+We add a new **IPPool** with the CIDR range, **10.0.0.0/16**.
+
+<pre>
+cat <<EOF | calicoctl apply -f -
+apiVersion: projectcalico.org/v3
+kind: IPPool
+metadata:
+name: new-pool
+spec:
+cidr: 10.0.0.0/16
+ipipMode: Always
+natOutgoing: true
+EOF
+</pre>
+
+Let’s verify the new IP pool.
+
+`calicoctl get ippool -o wide`
```

Comment:
I think this should be in triple backticks. 

e.g. 
\`\`\`
calicoctl get ippool -o wide
\`\`\`

Reply:
(and below as well)

Reply:
Fixed.